securedrop/manage.py

```diff
@@ -446,6 +455,15 @@ def get_args():
     return parser
 
 
+def set_add_admin_parser(subps, name):
+    parser = subps.add_parser(name, help='Add an admin to the application.')
+    parser.add_argument('--password', help='force password')
+    parser.add_argument('--username', help='force username')
+    parser.add_argument('--totp', action='store_true', help='force TOTP')
+    parser.add_argument('--otp-secret', help='force OTP secret')
```

Comment:
I understand that you're doing this so we can create a test user, but since these options will be exposed to current SecureDrop administrators, I don't think we want to give them the ability to override any of these (for their own good) - what about a one off script that adds the test user (directly using `sqlalchemy`)?

Reply:
See script in this commit 4774568324965cc02f4dae2035d1d1fef71ba655 - you could run this instead, let me know what you think

Reply:
I would also prefer this script as we will not expose anything to the admins. If we provide any command line option available, someone will try those and complain if something breaks.

Reply:
Good point @redshiftzero @kushaldas, will use this script :-)